app/src/main/res/layout/activity_login.xml

```diff
@@ -42,9 +44,11 @@
         android:layout_width="240dp"
         android:layout_height="65dp"
         android:autofillHints="password"
+        android:background="@drawable/input_bg"
         android:ems="10"
         android:hint="@string/password_hint"
         android:inputType="textPassword"
+        android:paddingLeft="10dp"
```

Comment:
please add a scaled size to text hints, in some screens doesn't shows the entire message, as you can see below, , also u can add some padding between the "Email" and email field and "Password" and password field for better looking
![Registration](https://user-images.githubusercontent.com/108461677/178915456-6c119622-61af-48a8-9c0c-917c4a105bc4.png)

Reply:
your issue here comes from the hardcoded text on width, just play with it and the text will not be cut anymore, no need to add scalable text (or at least not yet)